package.json

```diff
@@ -8,7 +8,8 @@
     "start:tsc": "ts-node src",
     "lint": "eslint ./src --ext .js,.ts,.jsx,.tsx",
     "lint:fix": "eslint ./src --fix --ext .js,.ts,.jsx,.tsx",
-    "gen:types": "ts-node src/utils/gen-types"
+    "gen:types": "ts-node src/utils/gen-types",
+    "add:chain": "ts-node src --dry-run --repo grants-contracts-upgradeable && cd contracts/evm && yarn && yarn typechain && NETWORK=$NETWORK npm run deploy && cd ../.. && ts-node src/utils/gen-chain-yaml --chain-name $NETWORK"
```

Comment:
the dry run will commit to the grants-contracts-upgradeable repo in the tmp folder, not our submodule